<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabPnjs</class>
 <widget class="QWidget" name="TabPnjs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Lista de PNJ:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListView" name="listaPersonajes">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="botonLimpiar">
        <property name="text">
         <string>Limpiar</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Respuesta:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QTextEdit" name="cajaRespuestas">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QPushButton" name="botonGenerarPNJ">
        <property name="text">
         <string>Generar PNJ...</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icon/cmd</normaloff>:/icon/cmd</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="botonObtener">
        <property name="text">
         <string>Actualizar lista</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icon/eye</normaloff>:/icon/eye</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="botonObtenerSeleccionado">
        <property name="text">
         <string>Obtener seleccionado</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Nuevo PNJ...</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icon/new</normaloff>:/icon/new</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>botonLimpiar</sender>
   <signal>clicked()</signal>
   <receiver>cajaRespuestas</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
